src/main/java/com/github/onsdigital/dpjavadurationparse/ParseDuration.java

```diff
@@ -0,0 +1,75 @@
+package com.github.onsdigital.dpjavadurationparse;
+
+import java.time.Duration;
+import java.time.format.DateTimeParseException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ParseDuration {
```

Comment:
Since this is a utility class with static methods, it's recommended to add a private constructor to avoid instance creations